.omc/kb/raw/src-lmstudio-openai-compat.md

```
@@ -0,0 +1,92 @@
+
+ Note: The following examples assume the server port is 1234 
+
+ Python Example 
+
+ from openai import OpenAI 
+
+ client = OpenAI( 
+ + base_url="http://localhost:1234/v1" 
+ ) 
+
+ # ... the rest of your code ... 
```

markdownlint-cli2 (0.23.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)